docs/source/technical_tutorials/dps_tutorial/dps_tutorial_demo.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<ul><li>sets the correct python environment for your code to run</li><li>the best way to execute your algorithm with a custom environment is to use <code>conda run</code>, as shown in this script(<code>conda run --live-stream --name dps_tutorial python ${basedir}/gdal_wrapper.py --input_file ${INPUT_FILENAME} --output_file output/${OUTPUT_FILENAME} --outsize ${REDUCTION_SIZE}</code></li><li> )</li></ul><p> </p><blockquote> )</blockquote><p>Do we need this <code>)</code> to be a separate bullet point?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Hm I do not see that extra bullet point in my browser...</p><p> </p>